src/osm/OsmiumHandler.cpp

```diff
@@ -191,17 +193,20 @@ void osm2rdf::osm::OsmiumHandler<W>::relation(
   if (_config.adminRelationsOnly && relation.tags()["admin_level"] == nullptr) {
     return;
   }
-  auto r = osm2rdf::osm::Relation(relation);
+  auto osmRelation = osm2rdf::osm::Relation(relation);
 #if BOOST_VERSION >= 107800
   if (_relationHandler.hasLocationHandler()) {
-    r.buildGeometry(_relationHandler);
+    osmRelation.buildGeometry(_relationHandler);
   }
 #endif  // BOOST_VERSION >= 107800
   if (!_config.noFacts && !_config.noRelationFacts) {
     _relationsDumped++;
 #pragma omp task
-    _dumpHandler.relation(r);
+    _factHandler.relation(osmRelation);
   }
+
+#pragma omp task
+  _geometryHandler.relation(osmRelation);
```

Comment:
Maybe we should task this away if boost is available as done for `osmArea.finalize();`?

Reply:
I agree, will change this :)